Create Benchmark test

Szacsi5104 opened the pull request:
Csináltam benchmark-olás céljából egy Benchmark "tesztet" (nem tudom pontosan hogy a test mappába kéne-e helyezni), ami pár általam választott algoritmus futási idejét írja ki a konzolra.

Szacsi5104 committed:
Create Benchmark tests

Csináltam benchmark-olás céljából egy egy Benchmark "tesztet" (nem tudom pontosan hogy a test mappába kéne-e helyezni), ami pár általam választott algoritmus futási idejét írja ki a konzolra.

kovbe11 commented:
Megnézem a SonarCloudon mivel van problémája. Valószínűleg a random a gond, úgyhogy ezzel nem lesz gond.

kovbe11 commented on tests/Benchmark.py:
```diff
@@ -0,0 +1,31 @@
+from algorithms.sort import (quick_sort, bubble_sort, gnome_sort, pancake_sort)
+
+
+def rand_list_fill(size):
```
generate_random_list szerintem egy beszédesebb név lehetne.

kovbe11 reviewed (changes requested):
Szép munka, viszont kérlek nevezd át a lista generáló függvényt arra amit írtam. Köszi!

Szacsi5104 committed:
Rename funct to generate_random_list

Bence kérésére átneveztem a rand_list_fill funkciót generate_random_list-re illetve kicseréltem a for ciklusnál az i változót _ ra ezzel jelezve hogy azt nem használom

kovbe11 commented on tests/Benchmark.py:
```diff
@@ -0,0 +1,31 @@
+from algorithms.sort import (quick_sort, bubble_sort, gnome_sort, pancake_sort)
+
+
+def generate_random_list(size):
```
Rendben lesz, köszönöm

kovbe11 reviewed (approved):
Sikerült javítani a kódot, engedélyezem
